src/main/scala/managehelpcontentpublisher/Eithers.scala

```diff
@@ -0,0 +1,9 @@
+package managehelpcontentpublisher
+
+object Eithers {
+
+  def seqToEither[E, A](eithers: Seq[Either[E, A]]): Either[E, Seq[A]] =
```

Comment:
Maybe 

```
eithers.partitionMap(identity) match {
      case (firstLeft :: _, _) => Left(firstLeft)
      case (_, rs) => Right(rs)
    }
```

Reply:
Thanks for the tip.  On reflection, should probably return `Either[Seq[E], Seq[A]]` to avoid losing any information.  That would also be easier with a `partitionMap`.  
Or if only interested in the first failure should fail fast and not continue before reaching this function.